dist/build_deploy.sh

```diff
@@ -8,10 +8,16 @@ IMAGE="quay.io/app-sre/cincinnati"
 IMAGE_TAG=$(git rev-parse --short=7 HEAD)
 PROJECT_PARENT_DIR=$ABSOLUTE_PATH/../
 DOCKERFILE_DEPLOY="$ABSOLUTE_PATH/Dockerfile"
+RELEASE_DIR="${PROJECT_PARENT_DIR}/target/x86_64-unknown-linux-musl/release"
+RELEASE_OUTPUT_DIR="${PROJECT_PARENT_DIR}/$(date +'%Y%m%d.%H%M%S')"
```

Comment:
this will lead to workspace growing with every new build, I would like to overwrite all generated files or clean-up after the build

Reply:
Ah, isn't the CI job spawned with a clean workspace?

Reply:
no, this one has static slaves.